mahout/src/main/scala/org/apache/zeppelin/mahout/utils/utils.scala

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
this little diddy finds the jars in the Mahout install.  Based heavily on code from https://github.com/apache/mahout/blob/master/spark/src/main/scala/org/apache/mahout/sparkbindings/package.scala#L169